src/pages/NewListing.jsx

```diff
@@ -339,6 +345,14 @@ const NewListing = () => {
   const haveEnoughZoom =
     wallet?.zoomBalance > numberOfSelectedCards * data?.zoomBurnFee
 
+  const { data: currentAllowance, isLoadingAllowance } = useGetZoomAllowanceQuery(
```

Comment:
I don't believe react-query exposes an `isLoadingAllowance` field. It should be `isLoading: isLoadingAllowance`

Reply:
YEP! Correct.

Good reviews for me.